File: openreview/journal/journal_request/process/comment_process.py
def process(client, edit, invitation):

    SUPPORT_GROUP = ''
    VENUE_ID = ''
    forum = client.get_note(edit.note.forum)

    full_name = forum.content['official_venue_name']['value']
    editors = forum.content['editors']['value']

    if edit.note.content['title']['value'] == 'New Recruitment Response':
        #if response did not change, return
        references = client.get_note_edits(note_id=edit.note.id)
        if len(references) > 1 :
            previous_reference = references[1]
            if edit.note.content['comment']['value'] == previous_reference.note.content['comment']['value']:
                return


        subject = f'A new recruitment response has been posted to your journal request: {full_name}'
        message = f'''Comment: {edit.note.content['comment']['value']}
        
To view the comment, click here: https://openreview.net/forum?id={edit.note.forum}&noteId={edit.note.id}'''

        recruitment_note = client.get_note(edit.note.replyto)
        client.post_message(subject, recruitment_note.signatures, message, invitation=f'{VENUE_ID}/-/Edit', signature=VENUE_ID)

    else:
        subject_eic = f'Comment posted to your journal request: {full_name}'
        message_eic = f'''A comment was posted to your journal request.

Comment title {edit.note.content['title']['value']}

Comment: {edit.note.content['comment']['value']}

To view the comment, click here: https://openreview.net/forum?id={edit.note.forum}&noteId={edit.note.id}'''

        client.post_message(subject_eic, editors, message_eic, invitation=f'{VENUE_ID}/-/Edit', signature=VENUE_ID)

File: openreview/venue_request/process/commentProcess.js
function(){
  var or3client = lib.or3client;

  var GROUP_PREFIX = '';
  var SUPPORT_GROUP = GROUP_PREFIX + '/Support';

  or3client.or3request(or3client.notesUrl + '?id=' + note.forum, {}, 'GET', token)
  .then(function(result) {

    var forumNote = result.notes[0];
    var prettySignature = note.signatures[0].startsWith('~') ? note.signatures[0].replace(/~|\d+/g, '').replace(/_/g, ' ') : 'Support';
    var commentTitle = note.content.title ? note.content.title : 'Comment by ' + prettySignature;

    var message = {
      groups: note.readers,
      ignoreGroups: [SUPPORT_GROUP],
      subject: 'Comment posted to your request for service: ' + forumNote.content.title,
      message: 'A comment was posted to your service request. \n\nComment title: ' + commentTitle + '\n\nComment: ' + note.content.comment + '\n\nTo view the comment, click here: ' + baseUrl + '/forum?id=' + note.forum + '&noteId=' + note.id + '\n\nPlease note that with the exception of urgent issues, requests made on weekends or US holidays can expect to receive a response on the following business day. Thank you for your patience!'
    };

    if (commentTitle.includes('Process Completed')) {
      return or3client.or3request(or3client.mailUrl, message, 'POST', token)
    }

    var support_message = {
      groups: [SUPPORT_GROUP],
      subject: 'Comment posted to a service request: ' + forumNote.content.title,
      message: 'A comment was posted to a service request. \n\nComment title: ' + commentTitle + '\n\nComment: ' + note.content.comment + '\n\nTo view the comment, click here: ' + baseUrl + '/forum?id=' + note.forum + '&noteId=' + note.id
    };

    return or3client.or3request(or3client.mailUrl, message, 'POST', token)
    .then(result => or3client.or3request(or3client.mailUrl, support_message, 'POST', token));
  })
  .then(result => done())
  .catch(error => done(error));

  return true;
};
